topics/sequence-analysis/tutorials/mapping/slides.html

```diff
@@ -977,6 +944,73 @@
 
 Not an easy task..
 
+---
+class: top
+
+## Mapping tools
+
+![Timeline of mapping tools](../../images/mapping/ngs_read_mappers_timeline.jpeg)
+
+
+.footnote[60+ different mappers, many comparison papers. Figure from [10.1093/bioinformatics/bts605](https://doi.org/10.1093/bioinformatics/bts605) ]
+
+???
+
+Many different tools available
+
+Different strengths and weaknesses, comparison table in link
+
+---
+class: top
+
+# Mapping tools
+
+<table style="width:100%; table-layout:fixed; font-size:0.8em;">
```

Comment:
is it possible to use a markdown table here? I think we can put the css if it's needed in the slides and apply a class

Reply:
Sure, I'll check it.

Reply:
Done :D